src/js/options.ts

```diff
@@ -44,6 +44,12 @@ export interface BaseReactNativeOptions {
   /** Enable scope sync from Java to NDK on Android */
   enableNdkScopeSync?: boolean;
 
+  /** Enable NDK on Android
+   *
+   * @default true
+   */
+  enableNdk?: boolean;
```

Comment:
`l`: I'm assuming this overrides `enableNdkScopeSync`? may make sense to switch the order

Reply:
yes, if `enableNdk` is `false`, `enableNdkScopeSync` is ignored